### ISSUE TITLE
Show Deck Win/Loss Info

### ISSUE THREAD

**Issue by jceddy** [jceddy/DailyArenaDeckAdvisor#12]
Show a deck's Win/Loss (and total games) record on its detail page and/or in the tab.

This is fairly easy for deck information coming from mtgarena.pro, as it's already included in the json that the scraper grabs, for archetypes pulled down from mtggoldfish are more difficult, though. There is some win/loss record information available, but it's not nearly a thorough or easy to get at as what mtgarena.pro has.

I'm currently investigating pulling more information from mtggoldfish for this and other issues, we'll see where it goes.

**Comment by jceddy** [comment]
This is the change that was voted to the top on the last release blog post, so I will devote time to working on it this week.

**Comment by jceddy** [comment]
Commit https://github.com/jceddy/DailyArenaDeckAdvisor/commit/d6b796905f30f21483f3663a611889acfc1f6a42 adds handling for this on the client side. So far this data has only been added for Arena Standard decks on the server side.

**Comment by jceddy** [comment]
The code added for this feature is working as expected. The server-side data is there for the Arena Standard format, as well as for the Standard format for decks where I was able to get some win/loss data from mtggoldfish. It's probably not as useful for the Standard decks, though, since the stats are all *match* statistics from tournaments where the decks did well and where the best versions of the decks were played by the top players...which means the win percentages are going to be skewed much higher than they would be if looking at single game statistic, and if included games played by all players (rather than just top tournament performers).

Anyway, that's the best we've got for now until we get a better source for Standard/Brawl data, or we have a bigger effort to manually generate/input our own win/loss stats. I'm considering this one closed.
